frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/backgroundables/FetchBackgroundable.java

```diff
@@ -35,37 +33,20 @@ public FetchBackgroundable(Project project,
       String remoteName,
       String refspec,
       String taskTitle,
-      @Nullable @Untainted String taskSubtitle,
       @Untainted String failureNotificationText,
       String successNotificationText) {
     super(project, taskTitle, /* canBeCancelled */ true);
     this.project = project;
     this.gitRepository = gitRepository;
     this.remoteName = remoteName;
     this.refspec = refspec;
-    this.taskSubtitle = taskSubtitle;
     this.failureNotificationText = failureNotificationText;
     this.successNotificationText = successNotificationText;
   }
 
-  public FetchBackgroundable(Project project,
-      GitRepository gitRepository,
-      String remoteName,
-      String refspec,
-      String taskTitle,
-      @Untainted String failureNotificationText,
-      String successNotificationText) {
-    this(project, gitRepository, remoteName, refspec, taskTitle, /* taskSubtitle */ null, failureNotificationText,
-        successNotificationText);
-  }
-
   @Override
   @UIThreadUnsafe
   public void run(ProgressIndicator indicator) {
-    if (taskSubtitle != null) {
-      // This method set a text under a progress bar (despite docstring)
-      indicator.setText(taskSubtitle);
```

Comment:
Hmmm okay... so subtitle is no longer needed?

Reply:
I was the only place where we set indicator text and the message was almost the same as the task title - this thing was  redundant